app/src/main/java/com/parishod/watomatic/fragment/MainFragment.java

```diff
@@ -274,6 +273,19 @@ public void onClick(View v) {
         }
     }
 
+    private boolean isAppUsedSufficientlyToAskRating(){
+        DbUtils dbUtils = new DbUtils(mActivity);
+        try {
+            long appFirstInstallTime = mActivity.getPackageManager().getPackageInfo(BuildConfig.APPLICATION_ID, 0).firstInstallTime;
+            if(System.currentTimeMillis() - appFirstInstallTime > 2 * 24 * 60 * 60 * 1000L && dbUtils.getNunReplies() >= MIN_REPLIES_TO_ASK_APP_RATING){
```

Comment:
Approved and merging but we should perhaps do time from first auto reply in the database